tools/iceoryx_build_test.sh

```diff
@@ -353,13 +361,13 @@ for COMPONENT in $COMPONENTS; do
     case "$1" in
         "unit")
             if [ ! -f testresults/"$COMPONENT"_ModuleTestResults.xml ]; then
-                echo "xml:""$COMPONENT""_ModuletestTestResults.xml not found!"
+                echo "xml: ${COMPONENT}_ModuletestTestResults.xml not found!"
```

Comment:
Now there is an extra space which wasn't there. Is this correct and/or necessary?

Reply:
Removed it